README.md

```diff
@@ -1,23 +1,18 @@
 # node-js-playwright-browserstack
-
-[Playwright](https://playwright.dev/docs/intro) Integration with BrowserStack.
+This repo contains samples for running [Playwright](https://playwright.dev/docs/intro) tests on BrowserStack using the browserstack-node-sdk.
 
 ![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
 
 ## Setup
 
-* Clone the repo and run `cd node-js-playwright-browserstack`
-* Set `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY` as environment variables with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings)
-* Run `npm i`
+* Clone the repo `git clone -b sdk https://github.com/browserstack/node-js-playwright-browserstack.git` and run `cd node-js-playwright-browserstack`.
+* Set `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY` as environment variables with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings) or update the same in `browserstack.yml` file.
+* Run `npm i` to install the dependencies.
 
 ## Running your tests
 
-- To run a sample test, run `npm run sample-test`
-
-## Run tests on locally hosted websites
```

Comment:
Steps for running local steps

Reply:
right now the main command runs all 3 tests.. will sync f2f to disc the approach for demonstrating local vs normal tests..

Reply:
added